0000-relationship-between-protocols/0000-relationship-between-protocols.md

```diff
@@ -0,0 +1,88 @@
+
+## Perspective
+First of all, the diagram below shows the entire relationship between the protocols.
+
+![Perspective](images/perspective.svg)
+
+### Elements and Brief Explanations
+
+- Connections
+  - (A) BTP over WebSocket
+    - **Node** to **Node** connection
+    - In order to exchange payments, configuration and routing information, a node MUST have a WebSocket connection to another node.
+  - (B) Ledger specific connection
+    - **Node** to **Ledger** connection
+    - A node needs the means to settle its payments. Therefore the node has a connection to a ledger. The type of connection varies depending on the ledger.
+  - (C) SPSP over HTTPS
+    - **Application** to **SPSP Server** connection
+    - To determine payment information such as a shared secret, a destination address and so forth, an application connects to SPSP server over HTTPS.
+- Protocols and related specifications
+  - (1) BTP
+    - [Bilateral Transfer Protocol 2.0 (BTP/2.0)](../0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md)
+    - BTP is used for transferring ILP packets and messages that are used for settlements and so on between two nodes.
+  - (2) ILP
+    - [Interledger Protocol V4 (ILPv4)](../0027-interledger-protocol-4/0027-interledger-protocol-4.md)
+    - ILP is used for transferring payment, node configuration, and routing information.
+  - (3) STREAM
+    - [STREAM - A Multiplexed Money and Data Transport for ILP](../0029-stream/0029-stream.md)
+    - STREAM is used for transferring money and data bidirectionally from applications.
+  - (4) Application
+    - The extensible data area of STREAM packets is used arbitrarily by applications.
+  - (5) DCP
+    - [Interledger Dynamic Configuration Protocol (ILDCP) v1](../0031-dynamic-configuration-protocol/0031-dynamic-configuration-protocol.md)
+    - DCP is used in order to exchange node information such as an ILP address.
+  - (6) RBP
+    - Route Broadcasting Protocol
+    - RBP is used for transferring routing information to build routing tables.
+  - (7) SPSP
+    - [The Simple Payment Setup Protocol (SPSP)](../0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md)
+    - SPSP is used to determine payment information such as a shared secret, a destination address and so forth.
+  - (8) ASN.1
+    - [ASN.1 Project](https://www.itu.int/en/ITU-T/asn1/)
+    - ASN.1 is used in order to specify data structure (order, type, and length).
+    - Concrete packet structure of protocols is defined using ASN.1.
+      - [asn1](../asn1/README.md)
+  - (9) Canonical OER
+    - [Notes on OER Encoding](../0030-notes-on-oer-encoding/0030-notes-on-oer-encoding.md)
+    - ASN.1 doesn't specify its encoding rule. i.e. How packets are encoded in binary is defined separately.
+    - Canonical OER is one of the specifications that define the encoding rule.
+  - (10) ILP Address
```

Comment:
It might be worth including this under the ILP bullet point, since the addresses are kind of part of the core ILP

Reply:
I changed the structure of the section and the explanation a bit. What do you think?